modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+Clock Source                                       tsc           tsc              Warning   true
+Config file valid                                  true          true             Fatal     true
+Connections listen backlog size                    >= 4096       4096             Warning   true
+Data directory filesystem type                     xfs           xfs              Warning   true
+Data directory is writable                         true          true             Fatal     true
+Data partition free space [GB]                     >= 10         1755.29          Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity set     true          true             Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity static  true          true             Warning   true
+Dir '/var/lib/redpanda/data' nomerges tuned        true          true             Warning   true
+Dir '/var/lib/redpanda/data' scheduler tuned       true          true             Warning   true
+Free memory per CPU [MB]                           2048 per CPU  7659             Warning   true
+Fstrim systemd service and timer active            true          true             Warning   true
+I/O config file present                            true          true             Warning   true
+Kernel Version                                     3.19          5.15.0-1056-aws  Warning   true
+Max AIO Events                                     >= 1048576    1048576          Warning   true
+Max syn backlog size                               >= 4096       4096             Warning   true
+NIC IRQs affinity static                           true          true             Warning   true
+NTP Synced                                         true          true             Warning   true
+RFS Table entries                                  >= 32768      32768            Warning   true
+Swap enabled                                       true          true             Warning   true
+Swappiness                                         1             1                Warning   true
+Transparent huge pages active                      true          true             Warning   true
+```
+
+[[redpanda-lmr]]
+=== Redpanda's latest version
+
+Check that Redpanda is running on the xref:https://github.com/redpanda-data/redpanda/releases[latest maintenance release^] on every node.
+
+Example:
+```bash
+/usr/bin/redpanda --version
+```
+
+Output:
+```bash
+v23.3.9 - f0a24fa4b60d55bd075c291e213807b5dfa74c0a
+```
+
+[[cpu-memory]]
+=== Correct CPUs and Memory configured
+
+Ensure that the correct number of CPUs and memory are being used by Redpanda.
+
+Example:
+
+```bash
+journalctl -u redpanda | grep "System resources"
+```
+
+Output:
+
+```bash
+Mar 25 12:16:18 ip-172-31-10-199 rpk[3957]: INFO  2024-03-25 12:16:18,105 [shard 0:main] main - application.cc:350 - System resources: { cpus: 8, available memory: 55.578GiB, reserved memory: 3.890GiB}
```

Comment:
we'll need to do a kubectl version of this for looking at kubernetes based deployments. (See also my comment at bottom of this doc about what whether we end up with two different versions of this checklist based on deployment method)

alternatively, we need to find a better command that will work no matter the deployment model

Reply:
@hcoyote @tmgstevens We can create a separate version/tab for k8s users, but I suspect that will be a whole new PR/different checklist. We want to get this out right away and circle back to create a k8s checklist, or wait for that checklist from you or the k8s team. Please advise wrt how you would like to handle that.

Reply:
circling back is fine with me.